x-pack/plugins/security_solution/server/lib/detection_engine/rules/prepackaged_rules/mfa_disabled_for_google_workspace_organization.json

```diff
@@ -30,5 +30,5 @@
   ],
   "timestamp_override": "event.ingested",
   "type": "query",
-  "version": 9
+  "version": 11
```

Comment:
This was already bumped from `7` to `9` in the last `8.2 Rules PR` without any other content changes...

https://github.com/elastic/kibana/pull/129526/files#diff-ed42a5bb623a69ce0970acd893b7a743ba196664154e11ffe245d86d69245c8eR33

Reply:
Double bump is the result of manual bumps done from the following.

https://github.com/elastic/detection-rules/pull/1870/files